3D/senior-csharp-test/Assets/Cache/FileCacheStorage.cs

```diff
@@ -52,218 +28,121 @@ private void InitCache()
 			return;
 		}
 
-		if( TryLoadCacheEntries() )
-		{
-			CheckCacheVersion();
-			ClearExpiredEntries();
-		}
-		else
-			DeleteAll();
+		ClearExpiredCacheFiles();
 	}
 
-	private bool TryLoadCacheEntries()
+	private void ClearExpiredCacheFiles()
 	{
-		string filePath = _cachePath + "/" + _fileName;
+		List<string> expiredfilePath = GetExpiredFilePaths();
```

Comment:
#niptuck if you keep this method
```suggestion
		List<string> expiredfilePaths = GetExpiredFilePaths();
```

Reply:
didn't keep it :)